sdk/core/Azure.Core/src/ChainingClassifier.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Core
+{
+    internal class ChainingClassifier : ResponseClassifier
+    {
+        private ResponseClassificationHandler[]? _handlers;
+        private ResponseClassifier _endOfChain;
+
+        public ChainingClassifier((int Status, bool IsError)[]? statusCodes,
+            ResponseClassificationHandler[]? handlers,
```

Comment:
Is the precedence of statusCodes and handlers explicitly documented somewhere?

Reply:
Only partially: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/RequestOptions.cs#L100

Will improve.